### ISSUE TITLE
Calendar app stop syncing.

### ISSUE THREAD

**Issue by a-i-m-a-r** [ubports/calendar-app#143]
Since upgrading to xenial, the calendar app does not sync new events created in the phone nor in the desktop.
I tried unlinking the caldav acount from calendar app and linking it again, but without luck.
I uninstalled the calendar app and installed it again. The first syncing worked and imported the new events created in the desktop, but no more sync is done afterwards.
After pressing the syncing button, it is disabled for a few seconds and enabled again. It is so short that I am sure that no sync has been done. And if I check the new events, there are not.

**Comment by a-i-m-a-r** [comment]
If I am not wrong, a log folder/file should be created in each sync. I checked the cache folder and if I try to sync no folder/file is created.

**Comment by a-i-m-a-r** [comment]
Maybe this may help.
http://paste.ubuntu.com/p/6GgB7cV7mY/
I got it from logviewer app.

**Comment by mardy** [comment]
Hi @a-i-m-a-r! Could you please check if you have some files under `~/.cache/syncevolution/` and upload them somewhere?
Just make sure to replace any sensitive data (contact names, e-mail addresses, etc.) that should appear in the logs, if you care about your privacy. :-)

**Comment by a-i-m-a-r** [comment]
There are some folders and the last one is the one created when I uninstall/reinstall the app (generic_+caldav_+13-2018-11-06-16-01):
There are two files that can be checked here: https://ehubox.ehu.eus/s/WcDzC5bBCnZdfK7
I have erased the domain of the server.

**Comment by mardy** [comment]
Thanks @a-i-m-a-r, that's interesting! The only date mentioned in the logs is November 6th; does it mean that you haven't tried to synchronize the calendar since then?
If not, then it seems that syncevolution is not being invoked at all; it's anyway a good starting point for an investigation. :-)

**Comment by a-i-m-a-r** [comment]
That is what I supposed, since the few seconds that the button "sync" is disabled is too short.
Every time I try to sync, nothing happens and no log is created in the cache folder.
So I think you are right and syncevolution is not invoked.. Does anybody have also similar issues?

**Comment by hummlbach** [comment]
When did you upgrade to xenial? 6th of November? :)

**Comment by a-i-m-a-r** [comment]
No.
I upgraded to xenial the 23rd of october. I have sync folders created regularly in the cache folder until that day.
Since then I could not sync again.
The one of the 6th of november was created after I uninstalled and reinstalled the app. That first sync was correct, but it cannot be synced anymore.

**Comment by hummlbach** [comment]
- so if you reinstall the app now, does it the sync then work again once for the first time?
- I don't know if @mardy wants to keep your current state for investigation purposes, but i was interested if deleting the evolution dot files (`.config/evolution`, `.cache/evolution`, `.local/share/evolution`) manually would solve your problem. But be careful to not lose calendar/contacts data only stored locally! 
- If you didn't play around with devel channel and beta releases and this is a general problem, then we have to solve this somehow, since we don't want to advise everyone to remove their data manually or something similar...

**Comment by a-i-m-a-r** [comment]
As suggested by @hummlbach, yesterday I reinstalled the calendar app, but I did as follows:
1.- I deleted the genericCaldav account in "settings"
2.- I deleted the cache, data and config files through UT Tweak Tool.
3.- I uninstalled the calendar app.
4.- I rebooted the phone
5.- I installed the calendar app
6.- I created the caldav account
7.- I synced ... and it worked, And still does. It syncs correctly.
I do not what happened during the xenial update. When I updated the phone asked to update also the apps, but I went offline and did it later manually. Many that was the origin of the problem.
Last time (6th november) I uninstalled/reinstalled the app, I did not follow steps 2 and 4. And I am not sure about the order of 1 - 3 and 5 - 6. I do not know it that could be significant.
